guides/lumen-supply-metrics.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Lumen Supply Metrics
+---
+
+[SDF’s Dashboard API endpoint](https://dashboard.stellar.org/api/v2/lumens) will always have the up-to-the-second totals for the essential numbers around lumens. This guide explains important supply metrics like Original Supply, Total Supply, and Circulating Supply entailed in that data. 
+
+## Dashboard API
+
+As of December 12, 2019, the Dashboard API shows:
+
+```json
+{
+  "updatedAt": "2019-12-12T21:07:23.480Z",
+  "originalSupply": "100000000000",
+  "inflationLumens": "5443902087.3472865",
+  "burnedLumens": "55442098181.3755700",
+  "totalSupply": "50001803905.9717165",
+  "upgradeReserve": "414310437.8111675",
+  "feePool": "1807341.5766261",
+  "sdfMandate": "29635032570.5297368",
+  "circulatingSupply": "19950653556.0541861",
+  "_details": "https://www.stellar.org/developers/guides/concepts/lumen-supply-metrics.html"
+}
+```
+
+## Definitions
+
+### originalSupply
```

Comment:
Nit: since these are object keys, it might be better to present them as
```suggestion
### `originalSupply`
```
Minor point though, not something I feel strongly about.

Reply:
i wonder what that will look like in the current docs - combining a H3 with a code snippet. let's see how it builds as is to reduce chances of errors for now?